package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "4.8.0",
+    "version": "4.7.0",
```

Comment:
Should we update templates in the generator?

Reply:
So, the generator looks correct. The current Sage People version is 4.8, as generated before. I'm not sure how you got version 4.7 of the API to build off of https://developer.sage.com/people/reference/

Reply:
seems like I took the wrong api description file. Re-generated with 4.8.